R/model_get_contrasts.R

```diff
@@ -11,7 +19,7 @@ model_get_contrasts <- function(model) {
 #' @rdname model_get_contrasts
 model_get_contrasts.default <- function(model) {
   tryCatch(
-    model$contrasts,
+    purrr::chuck(model, "contrasts"),
```

Comment:
I changed this to a chuck so it would through an error when the contrast object does not exist.  I don't think `model$contrasts` would have ever thrown an error.